cmd/cloudexec/clean.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func CleanJob(config config.Config, existingState *state.State, jobID int64, force bool) error {
-	prefix := fmt.Sprintf("job-%v", jobID)
+	prefix := fmt.Sprintf("job-%v/", jobID)
```

Comment:
Add a `/` to the end of the deletion prefix so that deleting `job-1` doesn't also delete from `job-10`, etc